test/tests

```diff
@@ -3,95 +3,124 @@
 source "$( pwd )/test/utils"
 source "$( pwd )/test/helpers"
 
-test::detect_legacy_default() {
-    # Test the buildpack detect script with a single simple .php file
-    test::utils::detect "legacy_default"
-    test::utils::assertCapturedSuccess
-    test::utils::assertCapturedEquals "PHP (classic)"
+# List of modules that we want to ship with every PHP setup.
+# For now, users can't specify a version for these modules, we just ship
+# what we have. Hence no specific versions constraints here.
+readonly default_modules=(
+    "apcu" "bcmath" "bz2" "Core" "ctype" "curl" "date" "dom" "exif" "fileinfo"
+    "filter" "gd" "hash" "iconv" "intl" "json" "libxml" "mbstring" "mongodb"
+    "mysqli" "mysqlnd" "openssl" "pcntl" "pcre" "PDO" "pdo_mysql" "pdo_pgsql"
+    "pdo_sqlite" "pgsql" "Phar" "posix" "readline" "redis" "Reflection"
+    "session" "shmop" "SimpleXML" "soap" "sockets" "SPL" "sqlite3" "standard"
+    "tokenizer" "xml" "xmlreader" "xmlwriter" "xsl" "Zend OPcache" "zip" "zlib"
+)
+
+
+test::legacy::defaults() {
+    # Test a deployment of a legacy app (not using Composer)
```

Comment:
Sure. I chose "legacy" because it's how we call it in our documentation :smile:
I'll open an issue to homogenize this to "classic" (and I'll modify the name in this PR).